packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md

```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
 
 <StackSelector snippet="extend-application" noSelector/>
 
-## Test the token inline hook
+## Preview and test the token inline hook
 
-The token inline hook is now ready for testing. You now have the following applications configured:
+The token inline hook is ready for preview and testing. You now have the following applications configured:
 
 * The Okta-Hosted-Login sample application (`samples-nodejs-express-4`) is ready to authenticate users from your Okta org.
 *  The external service (Glitch.com project) is ready with code to receive and respond to an Okta token inline hook call.
 *  The Okta org is set up to call the external service when a token inline hook is triggered by a user sign-in from the Okta-Hosted-Login sample application, and ready to receive a response.
 
 >**Note:** Make sure you have users assigned to your application and at least one user is part of the [Patients data store](/docs/guides/token-inline-hook/#check-against-the-data-store/) in your Glitch application.
 
+### Preview the token inline hook
+
+1. Navigate to Inline Hooks (**Workflow** > **Inline Hooks**) in your Admin Console.
```

Comment:
```suggestion
1. In the Admin Console, go to **Workflow** > **Inline Hooks**.
```
This is just a suggestion from infodev guidelines.

Reply:
Thanks, yes, I used that for updates to the same content on infodev. I'm fine to use this style.